packages/components/src/components/collapsible/arrow-button.tsx

```diff
@@ -93,7 +101,7 @@ const ArrowButton = ({ is_collapsed = false, position, onClick, title, handle_bu
             onClick={toggleExpand}
             onKeyDown={toggleExpand}
         >
-            {icon_arrow}
+            {!hide_collapsible_button && icon_arrow}
```

Comment:
do you think its easier logically if we do show_collapsible_button instead?

Reply:
Will apply, it's better to rename, you are right

Reply:
guys :) `hide` word is a quite commonly used word for booleans ) Double negation would be !has_no_button, but !hide_button is quite readable to me ))